ambry-store/src/main/java/com.github.ambry.store/TimeRange.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.store;
+
+/**
+ * A TimeRange is the range between two times in Ms
```

Comment:
It's in seconds, I will change this.
Regarding to the reason to lean towards one or the other, there are no obvious deciding factor. A conversion need to be made at one point or another since the returned time stamp for the APIs are also in seconds.